mesonbuild/minstall.py

```diff
@@ -243,14 +244,22 @@ def restore_selinux_contexts() -> None:
               'Standard output:', out,
               'Standard error:', err, sep='\n')
 
-def apply_ldconfig(dm: DirMaker) -> None:
+def apply_ldconfig(dm: DirMaker, libdir: str) -> None:
     '''
     Apply ldconfig to update the ld.so.cache.
     '''
     if not shutil.which('ldconfig'):
         # If we don't have ldconfig, failure is ignored quietly.
         return
 
+    if 'bsd' in platform.system().lower():
```

Comment:
DragonFly BSD doesn't have `BSD` in `uname -s` output, so this `dragonfly` here would fail to match despite inheriting `ldconfig` from FreeBSD.

Reply:
Argh! And mesonlib has a different function for every BSD, but nothing to check for "is in the BSD family".

Reply:
Unfortunately, this commit already landed in 0.60.2 as is. I had really wanted to wait for more confirmation, but the release was imminent...